wallet/src/de/schildbach/wallet/ui/invite/InvitationCreatedFragment.kt

```diff
@@ -51,15 +62,20 @@ class InvitationCreatedFragment : Fragment(R.layout.fragment_invitation_created)
         appCompatActivity.setSupportActionBar(toolbar)
 
 //        profile_picture_envelope.avatarProfile = requireArguments().getParcelable()
-        diaplayOwnProfilePicture() // just for the testing purposes
+        displayOwnProfilePicture() // just for the testing purposes
```

Comment:
Do we change this later in another story?

Reply:
I thought so, but now I think I was wrong. I'll remove this comment.